publicsuffixlist.js

```diff
@@ -520,7 +520,7 @@ const fromSelfie = function(selfie, decoder) {
         allocateBuffers(byteLength);
         decoder.decode(bufferStr, pslBuffer8.buffer);
     } else if (
-        selfie instanceof Object &&
+        selfie !== null && typeof selfie === 'object' &&
```

Comment:
Actually `selfie` should be a _required_ argument, which means this check should not exist. The code should happily throw an error if `selfie` has the wrong kind of value.

But for the time being I've made it closest to what it was.

Reply:
>Actually selfie should be a _required_ argument

If you agree, I'd like to remove this check entirely, but I'm a bit concerned that it might break something in the browser extension.

Reply:
Yes I think this can be removed.

Reply:
Done.